ods_ci/tests/Tests/500__jupyterhub/jupyterhub-user-access.robot

```diff
@@ -103,20 +103,15 @@ Set Custom Access Groups
 Check New Access Configuration Works As Expected
     [Documentation]    Checks if the new access configuration (using two custom groups)
     ...                works as expected in JH
-    Launch Dashboard   ocp_user_name=${TEST_USER.USERNAME}  ocp_user_pw=${TEST_USER.PASSWORD}
-    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    Launch Dashboard   ocp_user_name=${TEST_USER4.USERNAME}  ocp_user_pw=${TEST_USER4.PASSWORD}
```

Comment:
TEST_USER_4

Reply:
Uf, you're right, Berto; good catch:

```
TEST_USER:
  AUTH_TYPE: ldap-provider-qe
  USERNAME: ldap-admin1
```

I have it modified in my local properties settings, damn. 

Anyway, I've checked the cluster config for the `ldap-admin1` and the situation is same - it is part of the `dedicated-admins` group, which has a role of `cluster-admin` assigned. I'll update the description.

Not sure for how long this test been failing in which release this got changed.

Reply:
Oh, sorry, I didn't get it - I though it was related to our discussion that we resolved later; now I see it, will fix. Thanks guys!!!